programs/checkout/src/lib.rs

```diff
@@ -0,0 +1,392 @@
+use anchor_lang::prelude::*;
+use anchor_spl::{
+    associated_token::AssociatedToken,
+    token::{Mint, Token, TokenAccount},
+};
+
+use solana_program;
+use solana_program::{account_info::AccountInfo, program::invoke, system_instruction};
+
+use exhibition::program::Exhibition;
+use exhibition::{self, Exhibit};
+
+pub mod state;
+use state::heap::Heap;
```

Comment:
I'd probably rename this structs to what the represent (Orderbook, CheckoutQueue). The actual data structures should ideally be imported